Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE_description.md

```diff
@@ -0,0 +1,11 @@
+This integration was integrated and tested with version 1.0 of Prisma SASE API.
+
+## Configure Prisma SASE on Cortex XSOAR
+
+1. Navigate to **Common Services** > **Identity & Access** > **Servers & Services**.
```

Comment:
I assume this is done in the Prisma SASE UI?  We should say that.
In the Prisma SASE UI, navigate....